core/arch/arm/plat-zynqmp/drivers/aes.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) Foundries Ltd. 2021 - All Rights Reserved
+ * Author: Jorge Ramirez <jorge@foundries.io>
+ */
+
+#ifndef AES_H_
+#define AES_H_
+
+#include <types_ext.h>
+#include <tee_api_types.h>
+
+#define CSU_AES_KEY_SRC_KUP	0x0
+#define CSU_AES_KEY_SRC_DEV	0x1
```

Comment:
How about:
```C
enum csu_key {
        CSU_AES_KEY_SRC_KUP,
        CSU_AES_KEY_SRC_DEV
};
```
...then the `key` parameter to `aes_encrypt_data()` would be self-explanatory.